css/properties/alignment-baseline.json

```
@@ -206,7 +206,7 @@
               "opera": "mirror",
               "opera_android": "mirror",
               "safari": {
-                "version_added": "≤13.1"
+                "version_added": "≤5.1"
```

Assuming it was implemented at the same time as the property.

```suggestion
                "version_added": "≤4"
```